tests/fixtures/config/invalid-sql-file.config.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Naming nit on these config files. They have been moved to a directory called config so we can exclude config from their name.

Reply:
Yeah I saw the `fixtures/config` dir, just forgot to move this there after rebasing my branch, will do so now.